<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fota</class>
 <widget class="QMainWindow" name="fota">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>641</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Server</string>
     </attribute>
     <widget class="QPushButton" name="Server_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>151</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Server Website</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>221</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Features</string>
      </property>
     </widget>
     <widget class="QLabel" name="ShowSSID_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Your SSID</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="ShowSSID_textEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Path_lineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>431</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ShowFolderContent">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>171</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>ShowFolderContents</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Features</string>
     </attribute>
     <widget class="QPushButton" name="LED_Button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>LED</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FingerPrint_Button">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>91</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>FingerPrint</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
